### PR TITLE
Appveyor issue

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ init:
       Import-Module '..\appveyor-tool.ps1'
 
 install:
+  - cmd: set PATH=C:\Ruby27-x64\bin;%PATH%
```
Comment (msevestre) [inline appveyor.yml:14]: @Yuri05 Trying to set a new version of Ruby explicitely and reverting your changes from before

**Comment by msevestre** [comment]
@Yuri05 I think it's a bug in appveyor
https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-r/builds/41850668#L1800
see here

The variable is given already WITHOUT /. This is what comes from the env variable. I am going to write a note to appveyor to report a bug

**Comment by msevestre** [comment]
for now the path is hard coded

**Comment by Yuri05** [comment]
well, well...

**Pull request merged**